examples/examples.test.ts

```diff
@@ -33,7 +33,7 @@ function validateVL(spec: TopLevelExtendedSpec) {
 }
 
 function validateVega(spec: TopLevelExtendedSpec) {
-  const vegaSpec = compile(spec).spec;
+  const vegaSpec = JSON.parse(JSON.stringify(compile(spec).spec));
```

Comment:
@domoritz  -- any idea why this makes some validation passes?

Reply:
@AkshatSh -- this commit should be separated from other changes as it's a controversial change.

Reply:
This removes undefines and is not something we should do in tests. I am strongly against this change.

Reply:
Oh I see, what happens when you stringify something that is undefined?